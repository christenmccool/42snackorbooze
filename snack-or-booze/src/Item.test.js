import { render, asFragment, fireEvent, waitFor } from '@testing-library/react';
import Item from './Item';
import { MemoryRouter } from 'react-router-dom';
import {snacks} from './mockTestDb';
import '@testing-library/jest-dom/extend-expect';
import '@testing-library/jest-dom';


test('renders without crashing', () => {
    render(<MemoryRouter initialEntries={['/snacks/hummus']}><Item items={snacks} cantFind="/snacks" params="hummus"/></MemoryRouter>);
});

test("if matches snapshot", () => {
    const {asFragment, debug} = render(<MemoryRouter initialEntries={['/snacks/hummus']}><Item items={snacks} cantFind="/snacks" params="hummus"/></MemoryRouter>);
    expect(asFragment()).toMatchSnapshot();
    debug();
})

