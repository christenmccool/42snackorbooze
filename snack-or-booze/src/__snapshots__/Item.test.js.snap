// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`if matches snapshot 1`] = `<DocumentFragment />`;
